### ISSUE TITLE
[UX] adding a video to a playlist

### ISSUE THREAD

**Issue by bikepunk** [Chocobozzz/PeerTube#2754]
I feel like adding a video to a playlist is uneasy.
The only way I found is, while logged in, to watch a video and then click "Save" button (misleading name IMHO). Then only 3 playlist names are displayed with a search box, the existence of more playlists is unclear. If you want to add the video to a playlist which is not amongst the 3, you have to search a string of the name of the playlist you're expecting. 
This is tricky.

=> "add to playlist" function  UX shall be improved 

=> There sould be a way to add videos to a playlist from the playlist page (https://tube.aquilenet.fr/my-account/video-playlists). When I click "Edit" button on this page I was expecting to be able to add videos. It's currently not possible

**Additional information**
* PeerTube version or URL: https://tube.aquilenet.fr/
* Browser name/version: Firefox 68.8.0esr (64-bit)
* NodeJS version: don't know

* Link to browser console log if useful:
* Link to server log if useful (journalctl or /var/www/peertube/storage/logs/):

**Comment by rigelk** [comment]
The interface shows a scrollbar for > 5 playlists: I cannot reproduce your example.

"Save to playlist" is too long for a button name IMHO, and there is already a tooltip that feature that text on hover.

**Comment by bikepunk** [comment]
> The interface shows a scrollbar for > 5 playlists: I cannot reproduce your example.

a screenshot of my example below. 
I have 4 created playlists, only 3 are showing up, an no scrollbar here on 2.1.1 version :-/

![Peertube](https://user-images.githubusercontent.com/14958221/82059870-e4878880-96c6-11ea-9823-676bef83bde7.png)

**Comment by bikepunk** [comment]
> "Save to playlist" is too long for a button name IMHO, and there is already a tooltip that feature that text on hover.

I agree "Save to playlist" is too long ; but "Save" is misleading. There shall be a way to improve the UX, but I don't know how.

**Comment by rigelk** [comment]
@bikepunk the tooltip is here to disambiguate.

**Comment by rigelk** [comment]
on develop:

![Screenshot_2020-05-15](https://user-images.githubusercontent.com/6329880/82061193-b014cc00-96c8-11ea-8f41-6ae31f610af5.png)

**Comment by rigelk** [comment]
> May be "Playlist" with a "+" simbol over ?

@bikepunk that's the icon we're using

**Comment by bikepunk** [comment]
> > May be "Playlist" with a "+" simbol over ?
> 
> @bikepunk that's the icon we're using

Almost ! I suggest the replace "Save" by "Playlist". It's more explicit to me.

A perfect UX needs no tooltip (it currently helps though :-)

supergald to learn that the scollbar stuff in already on develop <3

**Comment by jimgroom** [comment]
I agree, this remains both a point of confusion (just call it playlist) and you have to search for any playlists other than the first 3, not intuitive for sure.
